python/sglang/srt/managers/scheduler.py

```diff
@@ -814,6 +823,10 @@ def event_loop_normal(self):
             batch = self.get_next_batch_to_run()
             self.cur_batch = batch
 
+            if batch:
+                for req in batch.reqs:
+                    trace_event("schedule", req.rid)
```

Comment:
wrap this into a function and only call it if tracing is enabled.
The principal: if tracing is not enabled, the overhead should just be a single if/else, not a for loop